src/internal/utils/io.go

```diff
@@ -18,10 +18,13 @@ const dotCharacter = 46
 
 var TempPathPrefix = "zarf-"
 
-func MakeTempDir(tmpDir string) (string, error) {
+func MakeTempDir(tmpDir string) string {
```

Comment:
@Noxsios I think we might have had a slight miscommunication earlier. It's not the end of the world if we use `message.fatal()` for now but I think we should do our best to move that to as early in the function hierarchy as reasonably possible.

Could you add back the error to this return?

Context:
Eventually (sooner than later hopefully..) we're going to refactor how we handle all errors across Zarf so that we are bubbling errors back to their calling function as reasonably as possible. This will be important to give the Zarf UI and more structured and predictable backbone for its API.

I might have misspoke earlier when I said instead of just using fatal logs that call `os.Exit()` we could call `panic()` (and then do extra logic with `defer` and `recover()`). While that might make sense in some places, I don't think we want that to be the norm.